source/partials/consulting/_hero.slim

```diff
@@ -1,5 +1,8 @@
+.u-navPlaceholder
```

Comment:
This is not supposed to be used here I believe. It's used with the nav partial only